gap/main.g

```diff
@@ -321,9 +320,7 @@ Chief := function( galoisField,mat,a,b,IsHPC )
                 if IsHPC then
                         if l-k = 0 then
                             TaskListClearUpR[j][l][1] := ScheduleTask(
-                                [   TaskListPreClearUp[j][l],
```

Comment:
Was the `...[j][l]` deleted by accident?

Reply:
You're right. Then please also mention that change in the commit message.

In principle it would better if this change were a separate commit, since this has nothing to do with the `dummyTask` changes. OTOH this is such a small change, if you only change the commit msg I'm fine with it.